manifests/metallb-frr.yaml

```diff
@@ -0,0 +1,611 @@
+apiVersion: policy/v1beta1
```

Comment:
Kustomize support is already in place (or at least, a kustomization.yaml file in the folder).
I wonder if we could have one default layer and one that overrides the speaker instead of two distinct manifests with duplicated resources.

Reply:
I think it's the right way to leverage Kustomize but would be more fitting in a later PR, since these manifests aren't actually usable now. Maybe once we actually get to a point where the FRR integration is functional and when we also change the Helm/dev-env manifests to include it. What do you think?

Reply:
Sounds legit, but mention that on top of the manifest that this is temporary and we are gonna clean it later on before the FRR integration is finalized (or as part of that), maybe even as part of the file name so no one is tempted to use it.

Reply:
done